code/modules/vending/_vending.dm

```diff
@@ -866,8 +866,12 @@ GLOBAL_LIST_EMPTY(vending_products)
 			if(icon_vend) //Show the vending animation if needed
 				flick(icon_vend,src)
 			playsound(src, 'sound/machines/machine_vend.ogg', 50, TRUE, extrarange = -3)
-			new R.product_path(get_turf(src))
+			var/obj/item/V = new R.product_path(get_turf(src))
```

Comment:
You used a nice `vended` variable, why do you have to downgrade it to a single capital letter? `vended_item` would be even better

Reply:
Done!